clamav/model.py

```diff
@@ -152,20 +157,21 @@ class ClamAVScanEventClient(object):
     def __init__(self, response):
         self.client = sseclient.SSEClient(response)
 
-    def process_events(self) -> typing.Union[ClamAVScanResult, ClamAVError]:
+    def process_events(self) -> ClamAVScanResult:
         '''Process the events sent by our ClamAV service
 
         Our ClamAV service will send exactly one SSE-event which is of one of two types:
             1. An event of type 'error', with its data containing an error-code (a HTTP/1.1 Status
                 Code) and a message, in case an error was encountered when scanning.
             2. An event of type 'result' containing the scan result in its data.
 
-        This method blocks until an event is received and then returns an instance of the
-        corresponding model class.
+        This method blocks until an event is received and then either returns the scan result or
```

Comment:
nit: indicate that the method blocks until the scanning is done (either with a finding/no finding, or an error)